index.js

```diff
@@ -163,6 +165,12 @@ class ScmBase {
             checkoutConfig.prRef = o.build.prRef;
         }
 
+        const manifest = getAnnotations(o.job.permutations[0], 'screwdriver.cd/repoManifest');
+
+        if (manifest) {
+            checkoutConfig.manifest = manifest;
```

Comment:
Should also update the jsdoc for getCheckoutCommand for the optional field `manifest`